packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -664,6 +649,11 @@ export default defineComponent({
       clearIconProps,
     }
   },
+  // we will use this while we have problem with 'withConfigTransport'
```

Comment:
Isn't there the way to implement focus and blur?

Reply:
done it